lib/active_node/validations.rb

```diff
@@ -34,6 +34,18 @@ module Validations
     autoload :UniquenessValidator, 'active_node/validations/uniqueness'
 
     module ClassMethods
+      # Initializes and saves an object in a single step.
+      def create(attributes = nil, &block)
+        if attributes.is_a?(Array)
+          attributes.collect { |attr| create(attr, &block) }
+        else
+          object = new(attributes)
+          yield(object) if block_given?
+          object.save
+          object
+        end
+      end
+
```

Comment:
BTW, by checking in ActiveRecord's code, I realized this perhaps should not be in the validations module, but in the persistence module. It kinda makes sense. If you want I can move both methods over there as part of the PR.